src/UkModulusCheck/Helpers.fs

```diff
@@ -0,0 +1,33 @@
+namespace UkModulusCheck
+
+module internal Helpers =
+    open System
+    open System.IO
+
+    module List =
```

Comment:
Add xml docs for the functions in this file

Reply:
Those functions are only internal helpers and are not exposed anywhere in the API. Does it make sense to add the xml docs to them?